# Obstacle location, the world frame, and shape and size
uint16 id  # obstacle id
float64 x  # obstacle x-pose
float64 y  # obstacle y-pose
float64 width
float64 length
float64 height 
